training/src/main/resources/javascript/script.js

```diff
@@ -572,4 +572,40 @@ function showDate(){
     local_date = new Date();
 
     alert(local_date.toUTCString());
-}
\ No newline at end of file
+}
+
+function checkFormAddition(){
+    var a = document.getElementById("a");
+    var b = document.getElementById("b");
+    var result = document.getElementById("result");
+    var r = parseInt(a.value) + parseInt(b.value);
+    result.value = r;
+}
+
+function checkFormSubtraction(){
+    var a = document.getElementById("a");
+    var b = document.getElementById("b");
+    var result = document.getElementById("result");
+    var r = parseInt(a.value) - parseInt(b.value);
+    result.value = r;
+}
+
+function checkFormMoltiplication(){
+    var a = document.getElementById("a");
+    var b = document.getElementById("b");
+    var result = document.getElementById("result");
+    var r = parseInt(a.value) * parseInt(b.value);
+    result.value = r;
+}
+
+function checkFormDivision(){
+    var a = document.getElementById("a");
+    var b = document.getElementById("b");
+    var result = document.getElementById("result");
+    var r = parseInt(a.value) / parseInt(b.value);
+    result.value = r;
+}
```

Comment:
These functions are wrong, you should do it in a single one based on the symbol, like a calculator